README.rst

```diff
@@ -66,30 +65,33 @@ Here is a slightly fancier example:
 
    >>> class fancy(Decorum):
    ...     def init(self, arg=None):
+   ...         super(fancy, self).init()
```

Comment:
This use of super() is not mandatory here since parent class does nothing in init(). But I think it is a good practice.